app/forms/collection_edit_form.rb

```diff
@@ -21,6 +27,8 @@ def self.model_attributes(attrs)
     attrs.dup.each do |key, value|
       if value.is_a? Array
         attrs[key].map! { |v| to_hash(v) }
+      elsif value.nil? || value.to_s.strip.blank?
```

Comment:
Whatever you think is best. I agree it seems like the upstream super call "should" clean those empty attributes out. But, it also looks like between the super call and the current nil check additional attributes are getting added. Perhaps that's the source?

In any event, I'm completely good leaving your check in for now, or whatever you think best, since there's certainly nothing wrong with making sure we don't insert empty data (I hope) :)